packages/graphback-codegen-client/src/templates/gqlTemplates.ts

```diff
@@ -35,7 +35,7 @@ export const findAllQuery = (t: GraphQLObjectType) => {
 export const findQuery = (t: GraphQLObjectType) => {
   const fieldName = getFieldName(t.name, GraphbackOperationType.FIND)
 
-  return `query ${fieldName}($fields: ${t.name}Input!, $limit: Int, $offset: Int) {
+  return `query ${fieldName}($fields: ${t.name}Fields!, $limit: Int, $offset: Int) {
```

Comment:
In terms of names WDYT about:

`${t.name}Data` for creation
`${t.name}Filter for query and mutation filtering

Reply:
Agreed - data is more clear and fields was always wrong :D